index.js

```diff
@@ -1,38 +1,155 @@
+          cell1.value = 0;//state of the cell1
+
+          const pos1 = player1.gameBoard[x][y];
+          const pos2 = player2.gameBoard[x][y];
+
+          let lives1 =document.getElementById('ships_player1');
+          let lives2 = document.getElementById('ships_player2');    
+                  
+          lives2.textContent = `${player2.shipsCount}`;
+          lives1.textContent = `${player1.shipsCount}`;
+
+          cell1.addEventListener( 'click', (e) =>
+             {
+              if(player1.shipsCount !== 0 && player2.shipsCount !== 0)
+              {
+                if(turn.textContent === `${player2.player2_Name}`)
+                {
+                  let cell1 = e.target; 
+                  cell1.style.backgroundColor = "purple";
+                  turn.textContent = `${player1.player1_Name}`;
+                  if(pos1 == 1)
+                  {
+                    player1.shipsCount--;
+                    lives1.textContent = `${player1.shipsCount}`;
+                    if(player1.shipsCount == 0)
+                    {
+                      turn.textContent = `Congratulationes ${player2.player2_Name.toUpperCase()}!! you win!`;
+                      alert(`Congratulationes ${player2.player2_Name.toUpperCase()}!! you win!`);
+                    }
+                  } 
+                  cell1.textContent = `${pos1}`;  
+                } 
+                  else 
+                  {
+                    alert(`It is ${player1.player1_Name}'s turn!`)
+                  }
+              }
+            });
+
+          cell2.addEventListener( 'click', (e) =>
+          {
+            if(player1.shipsCount !== 0 && player2.shipsCount !== 0)
+            {
+              if(turn.textContent === `${player1.player1_Name}`)
+              {
+                let cell2 = e.target;
+                cell2.style.background ="purple";
+                cell2.textContent = `${pos2}`;
+                turn.textContent = `${player2.player2_Name}`;
+
+                if(pos2 == 1)
+                {
+                  player2.shipsCount--;
+                  lives2.textContent = `${player2.shipsCount}`;
+                  if(player2.shipsCount == 0)
+                  {
+                    turn.textContent = `Congratulationes ${player1.player1_Name.toUpperCase()}!! you win!`;
+                    alert(`Congratulationes ${player1.player1_Name.toUpperCase()}!! you win!`);
+                  }
+                } 
+              } 
+                else
+                {
+                  alert(`It is ${player2.player2_Name}'s turn!`)
+                }
+            }
+              
+          });
+
+          li1.appendChild(cell1); //adding each cell into the row number x
+          li2.appendChild(cell2);
+        }
+
+      board_Player1.appendChild(li1); //adding each row into the board
+      board_Player2.appendChild(li2);
     }
+  }
+
+  addShips();
+  document.getElementById('reset').addEventListener('click', (e) => {
```

Comment:
event (e) is not being used, then, it is not necessary to pass the event as an argument.

document.getElementById('reset').addEventListener('click', () => {

Reply:
✔️